drivers/serial/uart_nrfx_uarte.c

```diff
@@ -1580,6 +1592,19 @@ static void uarte_nrfx_set_power_state(const struct device *dev,
 		}
 #endif
 		if (nrf_uarte_event_check(uarte, NRF_UARTE_EVENT_RXSTARTED)) {
+#ifdef UARTE_INTERRUPT_DRIVEN
+			if (data->int_driven) {
+				if (nrf_uarte_int_enable_check(uarte,
+						NRF_UARTE_INT_ENDRX_MASK)) {
+					data->int_driven->rx_irq_enabled = true;
+					nrf_uarte_int_disable(uarte,
+						NRF_UARTE_INT_ENDRX_MASK);
+				} else {
+					data->int_driven->rx_irq_enabled =
+									false;
```

Comment:
Could we somehow avoid such crazy indentation? Maybe using something like:
```
				data->int_driven->rx_irq_enabled =
					nrf_uarte_int_enable_check(uarte
						NRF_UARTE_INT_ENDRX_MASK);
				if (data->int_driven->rx_irq_enabled) {
					nrf_uarte_int_disable(uarte,
						NRF_UARTE_INT_ENDRX_MASK);
				}
```

Reply:
done